ceph-dashboard-pull-requests/config/definitions/ceph-dashboard-pull-requests.yml

```diff
@@ -30,9 +30,10 @@
           allow-whitelist-orgs-as-admins: true
           org-list:
             - ceph
+          white-list-labels: "dashboard"
           trigger-phrase: 'jenkins test dashboard'
           skip-build-phrase: '^jenkins do not test.*'
-          only-trigger-phrase: true
+          only-trigger-phrase: false
```

Comment:
this is "false" by default, hence can be removed.

Reply:
Okay, I can remove it - I left it there because I've seen it is set to ``false`` e.g. for the [ceph-pull-requests](https://github.com/ceph/ceph-build/blob/master/ceph-pull-requests/config/definitions/ceph-pull-requests.yml#L35) Jenkins config as well.